app/js/actions/labOrders/labOrderableAction.js

```diff
@@ -0,0 +1,35 @@
+import * as types from '../actionTypes';
+import axiosInstance from '../../config';
+import loading from '../loading';
+
+export const fetchLabOrderablesSuccess = orderables => ({
+  type: types.FETCH_LAB_ORDERABLE_SUCCESS,
+  orderables,
+});
+
+export const fetchLabOrderablesFailure = error => ({
+  type: types.FETCH_LAB_ORDERABLE_FAILURE,
+  error,
+});
+
+/**
+ * @function
+ * @returns {Object} - Object consisting lab orderables from API resource
+ * @throws {Object} - when an error occurs
+ */
+export const getLabOrderable = () => (dispatch) => {
+  dispatch(loading('FETCH_LAB_ORDERABLE', true));
+  return axiosInstance.get('/concept/da006137-88ca-4d11-ae58-8b4b439afdd6?v=full')
+    .then((response) => {
+      if (response.status !== 200) {
+        throw Error(response.statusText);
+      }
+      dispatch(loading('FETCH_LAB_ORDERABLE', false));
+      return response;
+    })
+    .then(({ data }) => dispatch(fetchLabOrderablesSuccess(data)))
+    .catch((error) => {
```

Comment:
You can put the `dispatch(fetchLabOrderablesSuccess(data))` in the first `then` block consequently you don't need a second `then` block.

Reply:
Yes, you are right...

Reply:
Done